examples/webgpu_mrt.html

```diff
@@ -108,10 +108,10 @@
 					const diffuse = scenePass.getTextureNode( 'diffuse' );
 					const emissive = scenePass.getTextureNode( 'emissive' );
 
-					const out = mix( output.renderOutput(), output, step( 0.2, viewportUV.x ) );
-					const nor = mix( out, normal, step( 0.4, viewportUV.x ) );
-					const emi = mix( nor, emissive, step( 0.6, viewportUV.x ) );
-					const dif = mix( emi, diffuse, step( 0.8, viewportUV.x ) );
+					const out = mix( output.renderOutput(), output, step( 0.2, screenUV.x ) );
```

Comment:
Should be `viewportUV.x`.

Reply:
I tried it with a smaller viewport, and in works in the webgpu_mrt.html example.

Reply:
Why? That is not the intention of the example, it would be to consider the entire screen not an area of ​​the viewport. Plus, we accustom users to using `screen UV` which should perform better.

Reply:
Set a viewport to sub-area of the full screen and try it. See https://github.com/mrdoob/three.js/issues/29162#issue-2471781557.

Reply:
> Set a viewport to sub-area of the full screen and try it. See https://github.com/mrdoob/three.js/issues/29162#issue-2471781557.

We can create an example for this soon, I would like to not use `viewportUV` when it is not necessary.